packages/framework/tests/Unit/Model/Order/Item/OrderItemTest.php

```diff
@@ -26,7 +26,9 @@ public function testTransportCannotBeSetForWrongType(): void
         $orderItem = $this->createOrderPayment();
 
         $this->expectException(WrongItemTypeException::class);
-        $orderItem->setTransport($this->createTransportMock());
+
+        $transportMock = $this->createTransportMock();
+        $orderItem->setTransport($transportMock);
```

Comment:
These kind of changes are no longer necessary

Reply:
That is right, fixed.